rcfiles/jointLklDM.rc

```diff
@@ -35,14 +35,18 @@ jointLklDM.plotsDir: plots
 # Comment out if you don't want to save or read the histograms but create them on the fly
 # (that is slower but safer because you make sure you use the proper histograms and not
 #  some pre-existing version created for a different set of IRFs or samples)
-jointLklDM.dNdEpSignalDir: DM/dNdEpSignal
+# Comment out for the channel 'branon' for now!!!
+# Comment out for the channel 'custom' always!!!
+#jointLklDM.dNdEpSignalDir: DM/dNdEpSignal
 
 ######################
 ANALYSIS CONFIGURATION
 ######################
 
-# channel to be analyzed (bb, tautau, mumu, WW, gammagamma, pi0pi0 or pi0gamma)
-jointLklDM.Channel: bb
+# channel to be analyzed (bb, tautau, mumu, WW, gammagamma, pi0pi0, pi0gamma, ee, branon or an arbitrary linear combination of them)
+# The linear combination must be written this form:'br_1*channel_1+...+br_n*channel_n'!
+jointLklDM.Channel: 1.0*bb+0.0*tautau
+#jointLklDM.Channel: bb
```

Comment:
Remove this line, single BRs an channels should be a particular case of the general, multiple BRs and channels. 
You have to allow that no BR value is provided for a given channel, in which case it should be interpreted as BR=1.

Reply:
Yes!